scripts/hail_batch/data_transfer/copy_densified_pca_images.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil cp -r "gs://cpg-tob-wgs-main/tob_wgs_hgdp_1kg_nfe_pca_densified/v0" $OUTPUT
```

Comment:
`$OUTPUT` will already have been created (for the metadata file), so I'd suggest instead:

```suggestion
gsutil cp -r "gs://cpg-tob-wgs-main/tob_wgs_hgdp_1kg_nfe_pca_densified/v0/*.png" $OUTPUT
```